src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx

```diff
@@ -7,12 +7,16 @@ import FlexRow from '../../../layout/FlexRow/FlexRow';
 import { ReactElement } from 'react';
 import { navbarOpenHeight } from '../../../../constants';
 import styled from 'styled-components';
+import { useThemeContext } from '../../../styles/Theme';
+import { Text } from '../../../..';
 
 export interface ProductTemplateV2 {
-  backgroundColor: string;
-  header: {
-    backgroundColor: string;
-    height: string;
+  titleClassName: string;
```

Comment:
All of the underlying elements can be targeted by utility classes to override them - supports Tailwind